packages/rspack/src/config/defaults.ts

```diff
@@ -319,7 +320,7 @@ const applyJavascriptParserOptionsDefaults = (
 	D(parserOptions, "importDynamic", true);
 	D(parserOptions, "worker", ["..."]);
 	D(parserOptions, "importMeta", true);
-	D(parserOptions, "inlineConst", inlineConst);
+	D(parserOptions, "inlineConst", production && inlineConst);
```

Comment:
Is it possible for Rspack to automatically disable the `inlineConst` when `optimization.usedExports` is not enabled?